contentcuration/contentcuration/frontend/shared/data/serverSync.js

```diff
@@ -267,30 +322,18 @@ async function syncChanges() {
     //   "errors": [],
     //   "successes": [],
     // }
-    if (requestPayload.changes.length != 0) {
-      socket.send(
-        JSON.stringify({
-          payload: requestPayload,
-        })
-      );
-    }
+
     const response = await client.post(urls['sync'](), requestPayload);
-    socket.onmessage = function(e) {
-      const data = JSON.parse(e.data);
-      console.log(data);
-      if (data.task) {
-        Task.put(data.task);
-      }
-    };
 
     try {
       await Promise.all([
-        handleDisallowed(response),
-        handleAllowed(response),
-        handleReturnedChanges(response),
-        handleErrors(response),
-        handleSuccesses(response),
-        handleMaxRevs(response, user.id),
+        handleDisallowed(get(response, ['data', 'disallowed'], [])),
+        handleAllowed(get(response, ['data', 'allowed'], [])),
+        handleReturnedChanges(get(response, ['data', 'changes'], [])),
+        handleErrors(get(response, ['data', 'errors'], [])),
+        handleSuccesses(get(response, ['data', 'successes'], [])),
+        handleMaxRevs(get(response, ['data', 'changes'], []), user.id),
```

Comment:
Looking at the former implementation of `handleMaxRevs`, it looks like we want to call it with:
```
get(response, ['data', 'changes'], [])
      .concat(get(response, ['data', 'errors'], []))
      .concat(get(response, ['data', 'successes'], []))
```